### PR TITLE
Use-case: caching module's configuration

### PR DESCRIPTION
**Pull request by Gamempire**
Avoid troubles in case of caching module's configuration

### CODE REVIEW

**Comment by gianarb** [comment]
Thanks for this improvement!

**Comment by Gamempire** [comment]
:+1:

File: docs/cache.md [hunk 1]
```diff
@@ -47,8 +47,43 @@ In this case you have to specify a custom factory in your `service_manager` conf
 ),
 ```
 
+Please be aware that you can't use *Closures* inside yout module configuration if you want to cache it. To avoid this problem, you have to use a Factory:
+```php
+// module.config.php
+'service_manager' => array(
+    'factories' => array(
+        'my_memcached_alias' => __NAMESPACE__ . '\Cache\MemcachedFactory'
```
Comment (danizord) [inline docs/cache.md]: Config files usually do not have namespaces. Can you replace `__NAMESPACE__` with `YourModule`?

Comment (danizord) [inline docs/cache.md]: Ah nvm, I did not saw that it was merged :D

**Pull request merged**